libcutils/Makefile

```diff
@@ -22,6 +22,7 @@ commonSources := \
 	strdup8to16.c \
 	record_stream.c \
 	process_name.c \
+	properties.c \
```

Comment:
The idea here was to have either properties.c for Android, or properties_nonandroid.c for the others, providing the same functions. Can we do without the properties.c addition (and changes)?